ansible/roles/ps4-controller/templates/ds4drv-service.j2

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=ds4drv daemon
+Requires=bluetooth.service
+After=bluetooth.service
+
+[Service]
+EnvironmentFile=/etc/donkeycar/ds4drv.env
+ExecStart=sudo /home/pi/env/bin/ds4drv --controller-limit 1 --led {{ ps4_contoller_led_color }}
```

Comment:
- [ ] You forgot a `v` in the `venv`
- [ ] You need to replace your variable by the environment variable : `--led $CONTROLLER_LED_COLOR `